src/main/java/com/laba/solvd/jdbc/utils/MyBatis.java

```diff
@@ -0,0 +1,46 @@
+package com.laba.solvd.jdbc.utils;
+
+import com.laba.solvd.jdbc.factory.IConnectionMethod;
+import org.apache.ibatis.io.Resources;
+import org.apache.ibatis.session.SqlSession;
+import org.apache.ibatis.session.SqlSessionFactory;
+import org.apache.ibatis.session.SqlSessionFactoryBuilder;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import static com.laba.solvd.jdbc.Main.connectionFactory;
+
+public class MyBatis implements IConnectionMethod {
+    private static final Logger logger = LogManager.getLogger(MyBatis.class.getName());
```

Comment:
please fix this in all classes 
```suggestion
    
    private static final Logger LOGGER = LogManager.getLogger(MyBatis.class.getName());
```